aten/src/ATen/NamedTensorUtils.cpp

```diff
@@ -69,7 +60,7 @@ static void check_for_misalignment(
     return;
   }
   auto it = std::find_if(other_names.begin(), other_names.end(),
-      [&](const Dimname& candidate) { return name.can_refer_to(candidate); });
+      [&](const Dimname& candidate) { return name == candidate; });
```

Comment:
I think you can use std::find() without the lambda?

Reply:
That's a good point.

Reply:
Addressed, that made the code a lot nicer. Thank you for catching it!